core/reportExporterOverview.py

```diff
@@ -22,6 +22,65 @@
     main_logger.critical("Correct report building isn't guaranteed")
     from core.defaults_local_config import *
 
+# Get errors history from Jenkins api
+def get_overview_reports_locations(engines, credentials, job_history_url, current_build_number="0", work_dir = ""):
+    try:
+        errors_history_data = {}
+
+        if(len(current_build_number) and int(current_build_number) > 0):
+            # Get overview reports urls from Job
+            job_data = {}
+            if credentials == "none":
+                job_data = requests.get('/'.join([job_history_url, "/api/json?tree=builds[url]"])).json()
+            else:
+                user_password = credentials.split(':')
+                job_data = requests.get('/'.join([job_history_url, "/api/json?tree=builds[url]"]), auth=HTTPBasicAuth(user_password[0], user_password[1])).json()
+
+            current_build_number = int(current_build_number)
+            build_nb_max = len(job_data['builds']) if len(job_data['builds']) < current_build_number else current_build_number
```

Comment:
Текущий билд будет всегда последним в нашем случае, поэтому лучше сразу присвоить current_build_number
```suggestion
            build_nb_max = current_build_number
```